stdlib/ysh/args.ysh

```diff
@@ -165,9 +165,19 @@ func parseArgs(spec, argv) {
 
   var value
   var found
+  var escape_remaining = false
   while (i < argc) {
     var arg = argv[i]
-    if (arg.startsWith('-')) {
+
+    if (escape_remaining) {
+      if (not spec.rest) {
+        error "Too many arguments, unexpected '$arg'" (code=2)
+      }
+
+      call rest->append(arg)
```

Comment:
Ah OK I see this error message and append logic is from below

I might want to put this in one place, but I can see that might be difficult with the current structure